pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -695,3 +743,54 @@ func removeCsv(	c client.Client, r *rest.Config) error{
 	log.Info("No clusterserviceversion for the operator found.")
 	return nil
 }
+
+func getNewUserNotificationEmails(c client.Client) (string, error) {
+
+	secret := &v1.Secret{}
+	secretParams := client.ObjectKey{
+		Namespace: "redhat-ods-operator",
+		Name:      "addon-managed-odh-parameters",
+	}
+
+	err := c.Get(context.TODO(), secretParams, secret)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("'%s'", string(secret.Data["notification-email"])), nil
+}
+
+func updateUserNotificationsEmails(oldAlertmanagerConfig string, newUserNotificationEmails string) string {
+
+	userNotificationregex := regexp.MustCompile(`- name: 'user-notifications'\n  email_configs:\n  - to: '.*com'`)
+	userNotificationMatch := userNotificationregex.FindAllString(oldAlertmanagerConfig, -1)
+
+	emailsRegex := regexp.MustCompile(`'.*com'`)
```

Comment:
Maybe I'm just lacking some context here...what's up with the ".*com" stuff?

Reply:
So the first regex matches the following
```
    - name: 'user-notifications'
      email_configs:
      - to: 'santa@presents.com,elf@presents.com '
```

The second regex matches `santa@presents.com,elf@presents.com` and replaces it with the new emails